content/200-concepts/100-components/02-prisma-client/058-transactions.mdx

```diff
@@ -257,6 +259,7 @@ await prisma.$transaction(
     // Code running in a transaction...
   },
   {
+    isolationLevel: Prisma.TransactionIsolationLevel.Serializable, // optional
     maxWait: 5000, // default: 2000
     timeout: 10000, // default: 5000
```

Comment:
Same order as they are introduced/documented above:
```suggestion
    maxWait: 5000, // default: 2000
    timeout: 10000, // default: 5000
    isolationLevel: Prisma.TransactionIsolationLevel.Serializable, // optional
```

Reply:
Maybe also mention here that the available options are defined below?

Reply:
I think this is ok as it is, as there's a link to the section above.